lib/fs.js

```diff
@@ -1585,6 +1585,10 @@ function encodeRealpathResult(result, options, err) {
   }
 }
 
+// This is removed from the fs exports in lib/module.js in order to make
+// sure that this stays internal.
+const realpathCacheKey = fs.realpathCacheKey = Symbol('realpathCacheKey');
```

Comment:
That would make sense but I wouldn’t consider that to be something urgent… by then we might have a working implementation in libuv.